File: solutions/20/sol.js
const getFactorial = (n) => {
  let factorial = 1n;
  for (let num = 2n; num <= n; num++) {
    factorial *= num;
  }
  return factorial;
}
const getSumDigits = (n) => {
  const nStr = n.toString();
  let sum = 0;
  for (const char of nStr) {
    sum += char * 1;
  }
  return sum;
}
const factorial = getFactorial(10);
const sum = getSumDigits(factorial);
console.log(sum);

File: solutions/34/sol.go
package main

import (
	"fmt"
)

func getFactorial(n int) int {
	result := 1
	for num := 2; num <= n; num++ {
		result *= num
	}
	return result
}

func main() {
	fmt.Println(getFactorial((1)))
	fmt.Println(getFactorial((4)))
	fmt.Println(getFactorial((5)))
	fmt.Println(getFactorial((9)))
}
